### PR TITLE
Feature: add okxwallet

### PR DESCRIPTION
**Pull request by okxwallet**

### CODE REVIEW

File: packages/wallets/injective/src/okx.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { Wallet as WalletType } from "@injectivelabs/wallet-ts";
+import { WalletState } from "@xlabs-libs/wallet-aggregator-core";
+import { InjectiveWallet } from "./injective";
+import { ConcreteWalletConfig } from "./types";
+
+export class OKXWallet extends InjectiveWallet {
+  constructor(config: ConcreteWalletConfig) {
+    super({
+      ...config,
+      type: "keplr" as WalletType,
```
Comment (M-Picco) [inline packages/wallets/injective/src/okx.ts:10]: Does the OKX wallet set the `window.keplr` object?

**Review by M-Picco — COMMENTED** [review]
I'd suggest leaving the Solana adapter out of this PR. I'll try to upgrade the package and dependencies to modernize it and use the solana wallet-standard instead of adapters.

File: packages/wallets/injective/src/okx.ts (hunk 1 above)
Comment (okxwallet) [reply packages/wallets/injective/src/okx.ts:10]: yes, okx wallet set window.keplr

**Comment by okxwallet** [comment]
> I'd suggest leaving the Solana adapter out of this PR. I'll try to upgrade the package and dependencies to modernize it and use the solana wallet-standard instead of adapters.

ok, we will do it.

File: packages/wallets/evm/src/okx/connector.ts [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+import { Chain } from "@wagmi/core/chains";
+import { getAddress } from "ethers/lib/utils.js";
+
+import type {
+  OKXWalletProvider,
+  InjectedConnectorOptions,
+  Address,
+} from "./types";
+
+export class OKXWalletConnector extends InjectedConnector {
```
[the remark this answers is not in the document]
Comment (okxwallet) [reply packages/wallets/evm/src/okx/connector.ts:11]: We don't want to conflict with other wallets

**Comment by M-Picco** [comment]
@okxwallet I took the liberty of cloning and updating the branch with the latest changes. You can see a [PR](https://github.com/XLabs/wallet-aggregator-sdk/pull/21) open for it.

Please let me know what you think before I merge it.

**Comment by okxwallet** [comment]
> @okxwallet I took the liberty of cloning and updating the branch with the latest changes. You can see a [PR](https://github.com/XLabs/wallet-aggregator-sdk/pull/21) open for it.
> 
> Please let me know what you think before I merge it.

thanks, if there has some okx need to do, please let me know.

**Comment by okxwallet** [comment]
> > @okxwallet I took the liberty of cloning and updating the branch with the latest changes. You can see a [PR](https://github.com/XLabs/wallet-aggregator-sdk/pull/21) open for it.
> > Please let me know what you think before I merge it.
> 
> thanks, the message you leave is right. it can be merged. if there has some okx need to do, please let me know.

**Pull request closed without merging**